src/components/App.jsx

```diff
@@ -1,7 +1,7 @@
 /* global cozy */
 import React, { useState, useEffect, useCallback } from 'react'
 import { PropTypes } from 'prop-types'
-import flow from 'lodash/flow'
+import { flow, get } from 'lodash'
```

Comment:
Nit: we prefer multiple lodash imports to keep bundle sizes small (https://nolanlawson.com/2018/03/20/smaller-lodash-bundles-with-webpack-and-babel/)

Reply:
In this case, I think the bundle will be small anyway since we are tree shaking. I think the perf gain would be more in dev mode.

Reply:
Yep, presumably on build times as well since we don't need to parse the whole lodash bundle with specific imports.